wp-youtube-lyte.php

```diff
@@ -80,6 +82,7 @@
 $lyteSettings['selSize']=$selSize;
 $lyteSettings['links']=get_option('lyte_show_links');
 $lyteSettings['file']=$wyl_file."?wyl_version=".$wyl_version;
+$lyteSettings['file_lazyload']=$wyl_file_lazyload."?wyl_version=".$wyl_version;
```

Comment:
also add $lyteSettings['lyte_use_internal_lazyloader'] and assign it with the value of the (default false) filter, so we can reuse that instead of checking the filter at multiple places?

Reply:
Done in latest commit.